server/test/handlers/inputSchedule.test.js

```diff
@@ -0,0 +1,72 @@
+const appointmentHandler = require('../../models/handlers/workerTimeslot');
+
+describe('testing invalid user schedule input', () => {
+    beforeEach(() => {
+        jest.resetModules(); // Clears any cache between tests.
+    });
+
+    test('rejection of null values for worker and school ID', async () => {
+        // Arrange
+        const testworkerId = null;
```

Comment:
For greater consistency, I would suggest naming the variables with lowerCamelCase, i.e., testWorkerId.

Reply:
Thank you @catai9 for your review! I will change the naming convention!